src/components/FeeExplainer/FeeExplainer.tsx

```diff
@@ -344,11 +346,11 @@ const feeExplainerCardBody = { base: 4, md: 8 }
 
 type FeeExplainerProps = CardProps
 
-export const FeeExplainer: React.FC<
-  FeeExplainerProps & { parameters: FeeCurveParameters }
-> = props => {
-  const { FEE_CURVE_NO_FEE_THRESHOLD_USD } = props.parameters
-  const votingPower = useAppSelector(selectVotingPower)
+export const FeeExplainer: React.FC<FeeExplainerProps & { feeModel: ParameterModel }> = props => {
```

Comment:
extend `FeeExplainerProps` on L:347

Reply:
I don't remember where the spread props landed as best practice, but could do `({ feeModel, ...props })` to destructure and only pass card props to the card

Reply:
> extend `FeeExplainerProps` on L:347

78d3380f7d

As for the second one, probably going to avoid it for the performance reasons @woodenfurniture previously noted